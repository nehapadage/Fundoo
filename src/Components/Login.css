*{
    /*box sizing border-box;*/
    /* font-family: 'Roboto',sans-serif; */
}

.mainBody{
    height: 100vh;
 
     display: flex;
     flex-direction: row;
     place-content: stretch center;
    
 }

 .adminMainDiv{

    min-height: auto;
    width:  300px;
    padding: 40px;
    background-color: white;
  border : 1px solid #dadce0;
   
  border-radius: 10px;;
    margin: auto auto;
     margin-top: 4em;
}

.adminMainDivService {
    min-height: auto;
    width: 300px;
    padding: 40px;
    background-color: white;
    border: 1px solid #dadce0;
    /* border: 1px solid black; */
    border-radius: 10px;
    margin: auto auto;
    /* margin-left: 10%; */

}

.combineDiv{
    width: auto;
    padding: 2%;
    margin: auto auto;
    display: flex;
    flex-direction: row;
    /* justify-content: space-between; */
    /* border: 1px solid black; */
}

/* .adminMainDiv1{

    min-height: auto;
    width:  300px;
    padding: 40px;
    background-color: white;
  border : 1px solid #dadce0;
  border-radius: 10px;;
    margin: auto auto;
} */


.adminMainDiv1 {
    /* min-height: auto;
    width: 380px;
    padding: 40px;
    background-color: #dadce0;
    border: 1px solid #dadce0;
    border-radius: 10px;
    margin: auto auto;
    margin-right: 20%; */


    min-height: auto;
    width: 380px;
    padding: 40px;
    background-color: #dadce0;
    border: 1px solid #dadce0;
    border-radius: 10px;
    margin: auto auto;
    margin-right: 20%;
}

#fundoo{
    font-size: xx-large;
    display: flex;
    /* margin-top: 30px; */
    flex-direction: row;
    justify-content: center;
}

#flabel{
    color:blue;
}

#ulabel{
    color:red;
}

#nlabel{
    color:#FFC300 ;
}

#dlabel{
    color:blue;
}

#olabel{
    color:green;
}

#o2label{
    color:red;
}

#login{
   
    font-weight: bold;
    display: flex;
  
    flex-direction: row;
    text-align: center;
    justify-content: center;
   
}

#text{
    /* font-family: 'Roboto',sans-serif; */
    margin-top: 7px;
    color: #5f6368;
    font-size: 14px;
}

.textfields{

    width: auto;
    /* height: 370px; */
    height: 300px;
}

#EmailPass{
  
    height: 8px;
    width: 270px;
} 

#forgotPass{
    color: #1a73e8;
    background-color: transparent;
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
  
}



#loginButton{
    background-color:#1a73e8 ;
    margin-left: 170px;
}

#cardfuns{
    display: flex;
    flex-direction: row;
    /* place-content: stretch center; */
    /* justify-content: space-evenly; */

}

/* @media only screen and (min-width: 765px) {
    .adminMainDivService {
        margin-top: 0em;
    }
    .combineDiv{
        margin-top: 0em;
    }
    
    
} */

/* @media only screen and (min-width: 375px) {
    .adminMainDivService {
        margin-top: 0em;
    }
    .combineDiv{
        margin-top: 0em;
    }
} */

@media only screen and (max-width: 1024px) {
  

        #cardfuns{
            display: flex;
            flex-direction: column;
            /* justify-content: center; */
            align-items: center;
            margin-left: -2em;
    
        }
 
    .mainBody{
       
         flex-direction: column;
         align-items: center;
    }
    /* .adminMainDivService {
        margin-left: 10%;
    } */
    .adminMainDiv1 {
        margin-left: 10%;
    }
    .combineDiv{
        width: 100%;
        flex-direction: column;
        align-items: center;
        margin-left: 2em;
    }
  
    /* .adminMainDivService {
        margin-top: 30em;
       
    } */
} 

@media only screen and (max-width: 450px) {

.combineDiv{
    width: 100%;
    flex-direction: column;
    align-items: center;
    margin-left: 2em;
}

    .adminMainDivService {
        margin-top: 32em;
        width: 70%;
        margin-left: 2px;
       
    }
    .adminMainDiv1 {
        width: 70%;
    }
    #EmailPass {
        width: 230px;
    }
    /* #log{
        display: flex;
        flex-direction: row;
        justify-content: center;
    } */
}




















.maindiv{
    background-color:white;
    text-align: center;
    padding: 50px;
    width: 300px;
    height: 400px;
    margin: 430px;
    margin-top: 30px;
    border: 2px solid;
    border-radius: 10px;
    border-color: grey;
}

#text{
    /* font-family: 'Roboto',sans-serif; */
    margin-top: 7px;
    color: #5f6368;
    font-size: 14px;
}


  


#login{
    font-style: initial;
    font-size: x-large;
    font-weight: bold;
}

.maindiv img{
    width: 400px; height: 200px;
    align-content: center;
    background-repeat: no-repeat;
}


#email{
    width: 250px; height: 20px;
}

#password{
    width: 250px; height: 20px;
}

#loginButton{
    height: 35px;
    width: 80px;
    margin-top: -50px;
    margin-left: 190px;
}

#registerButton{
    height: 35px;
    width: 250px;
}

#registerText{
    width: 300px; height: 30px;
    font-size: xx-small;
    /* font-variant-caps: all-small-caps; */
    
}

#forgetPassword{
    /* height: 30px; */
    width: 145px;
    margin-top: 0px;
    text-align: center;
    font-size: x-small;
    background-color: rgb(155, 140, 206);
    /* font-family: sans-serif; */
    margin-right: -127px;
}



#learnMore{
    color: #1a73e8;
    background-color: transparent;
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
}

#createAccount{
    color: #1a73e8;
    background-color: transparent;
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
}

.close{
    padding: 10px
}

/* @media only screen and (max-width: 400px) {

    .maindiv{
        width: 220px;
    }

    #email{
        width: 200px; height: 20px;
    }
    
    #password{
        width: 200px; height: 20px;
    }

    #loginButton{
        margin-left: 170px;
    }
}

@media only screen and (min-width: 320px) {

    .maindiv{
        width: 200px;
    }

    #fundoo{
        margin-left: 45px;
    }

    #email{
        width: 200px; height: 20px;
    }
    
    #password{
        width: 200px; height: 20px;
    }

    #loginButton{
        margin-left: 150px;
    }
}



 */
