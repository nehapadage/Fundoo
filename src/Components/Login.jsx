import React, { Component } from 'react'
import './Login.css'
import userService from '../services/userService'
import Card from '@material-ui/core/Card';
import TextField from '@material-ui/core/TextField';
import Button from '@material-ui/core/Button';
import { createMuiTheme, MuiThemeProvider } from "@material-ui/core";
import Snackbar from '@material-ui/core/Snackbar';
import CircularProgress from '@material-ui/core/CircularProgress';


const theme = createMuiTheme({
    overrides: {
        MuiButton: {
            root: {
                textTransform: "lowercase"
            }

        }
    }

});

const theme2 = createMuiTheme({
    overrides: {
        'MuiCard': {
            'root': {
                width: '180px',
                height: '220px',
                backgroundColor: '#acacac'
            },

        }
    }
})

const theme3 = createMuiTheme({
    overrides: {
        'MuiCard': {
            'root': {
                width: '180px',
                height: '220px',
                backgroundColor: '#f7bb00'
            },

        }
    }
})

const theme1 = createMuiTheme({
    overrides: {
        'MuiPaper': {
            'root': {
                width: '180px',
                height: '220px',
                zIndex: '10px',
            },
            'elevation1': {
                boxShadow: '0px 2px 1px -1px rgba(0,0,0,0.2), 0px 0px 1px 2px rgba(0,0,0,0.14), 0px 1px 3px 0px rgba(0,0,0,0.12)'
            },
            // 'hover':{
            //     transition:'box-shadow 280ms cubic-bezier(.4,0,.2,1)'
            // }
        }
    }
})


class Login extends Component {

    constructor(props) {
        super(props);

        this.state = {

            email: "",
            password: "",
            email1: "neha7@gmail.com",
            password1: "neha123",
            emailError: "",
            passwordError: "",
            flag: false,
            progress: false,
            array: []
        }
    }


    handlechangeall = (event) => {
        this.setState({ [event.target.name]: event.target.value })
    }

    validateEmail(email) {
        // var re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        var re = /^([a-z\d\.-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/
        return re.test(email);
    }

    validate = () => {
        let isError = false;

        const errors = {
            emailError: "",
            passwordError: ""
        };



        if (!this.validateEmail(this.state.email)) {
            isError = true;
            errors.emailError = "Requires valid email";
        }

        if ((this.state.password.length < 6) || (this.state.password.length > 12)) {
            isError = true;

            errors.passwordError = "Password length should greater than 6 and less than 12";
        }

       

        this.setState({
            ...this.state,
            ...errors
        });

        console.log("In validate----->" + this.state);


        return isError;
    };

    handleloginSubmit = (event) => {



        event.preventDefault();
        this.validate()
        const err = this.validate();

        if (!err) {
            var loginData = {};
            loginData.email = this.state.email;
            loginData.password = this.state.password
            if(localStorage.getItem('cartID')){
                loginData.cartId=localStorage.getItem('LoginCartId')
            }

            console.log("logindata--> ", JSON.stringify(loginData))






            //     const timer = setInterval(this.state.progress, 20);
            // return () => {
            //   clearInterval(timer);
            // };

            userService.login(loginData).then((res) => {
                console.log("respnse in login--> ", res)

                console.log("****respnse in login token is--> ", res.data.id)


                if (res.data.id) {

                    console.log("Flag-------------->", this.state.flag);

                    // alert("Login Successful-----");
                    localStorage.setItem('LoginToken', res.data.id);
                    localStorage.setItem('firstName', res.data.firstName)
                    localStorage.setItem('lastName', res.data.lastName)
                    localStorage.setItem('email', res.data.email)
                    localStorage.setItem('userId', res.data.userId)
                    localStorage.setItem('imageUrl', res.data.imageUrl)
                    localStorage.setItem('Grid', true)
                    //    localStorage.setItem('Drawer',false)



                    var path = '/dashboard/notes'
                    this.props.history.push(path)
                }


            }).catch((err) => {
                this.setState({ flag: true })
                console.log("error in login--> ", err)
            })

        }


        if (!err) {
            // clear form
            this.setState({
                email: "",
                emailError: "",
                password: "",
                passwordError: ""
            });
        }

        //for loading
        this.setState({ progress: true })

        this.timer = setTimeout(this.changeProgress, 700);
    }

    changeProgress = () => {
        this.setState({ progress: false })
    }

    handleCreateAccountClick = () => {
        console.log("createAccount button clicked..")
        var path = '/createAccount'
        this.props.history.push(path)
    }

    handleforgetPasswordSubmit = () => {
        console.log("forget password button clicked..")
        var path = '/forgetpassword'
        this.props.history.push(path)
    }

    handleClose = () => {
        console.log("Flag before", this.state.flag);

        this.setState({ flag: false })

        // console.log("Flag after", this.state.flag);
    };

    componentDidMount() {
        userService.getService().then(res => {
            console.log("Response in like question--->", res);
            this.setState({ array: res.data.data.data })

            // console.log("Array is",this.state.array[0].price);


        })
            .catch(err => {
                console.log("Error in get all data");
            })
    }


    render() {
        var styles = localStorage.getItem('cartID') ? "adminMainDivService" : "adminMainDiv"
        var styles1 = localStorage.getItem('cartID') ? "combineDiv" : null

        return (
            <MuiThemeProvider theme={theme}>
                <div className="mainBody">
                    <div className={styles1}>


                        <div className={styles}>
                            <div id="fundoo">
                                <label id="flabel">F</label>
                                <label id="ulabel">u</label>
                                <label id="nlabel">n</label>
                                <label id="dlabel">d</label>
                                <label id="olabel">o</label>
                                <label id="o2label">o</label>
                            </div>

                            <div>

                                <div id="login">Login </div>
                                <div id="text">Use your Fundoo Account</div>

                            </div>

                            <div className="textfields">
                                <div className="emailIdField">
                                    <TextField
                                        id="EmailPass"
                                        label="Email Id"
                                        type="email"
                                        name="email"
                                        value={this.state.email}
                                        autoComplete="email"
                                        margin="normal"
                                        variant="outlined"
                                        onChange={this.handlechangeall}
                                        errorText={this.state.emailError}

                                    />

                                </div>
                                <div style={{ fontSize: 12, color: "red" }}>
                                    {this.state.emailError}
                                </div>

                                <div className="PasswordField">
                                    <TextField
                                        id="EmailPass"
                                        label="Password"
                                        type="password"
                                        name="password"
                                        value={this.state.password}
                                        autoComplete="current-password"
                                        margin="normal"
                                        variant="outlined"
                                        onChange={this.handlechangeall}

                                    />
                                </div>


                                <div style={{ fontSize: 12, color: "red" }}>
                                    {this.state.passwordError}
                                </div>

                                <div>
                                    <Button id="forgotPass"
                                        onClick={this.handleforgetPasswordSubmit}>Forgot Password?</Button>
                                </div>
                                <div id="text">Not your computer? Use Guest mode to sign in privately.</div>
                                <div>
                                    <Button id="learnMore">Learn more</Button>
                                </div>
                                <div id="log">
                                    <Button id="createAccount"
                                        onClick={this.handleCreateAccountClick}
                                    >Create account
                                    </Button>
                                    <div>
                                        <Button
                                            id="loginButton"
                                            variant="contained"
                                            color="primary"
                                            onClick={this.handleloginSubmit}

                                        >
                                            Login
                                        </Button>
                                    </div>
                                    {this.state.progress ? <CircularProgress /> : null}

                                    <div>
                                        <Snackbar

                                            anchorOrigin={{
                                                vertical: 'bottom',
                                                horizontal: 'center',
                                            }}
                                            open={this.state.flag}
                                            autoHideDuration={6000}
                                            onClose={this.handleClose}

                                            message="Login Failed"


                                        />
                                    </div>

                                </div>
                            </div>

                            {/* here */}

                        </div>

                        {/* here */}
                        {localStorage.getItem('cartID') ?
                            <div className="adminMainDiv1">
                                <div>Service</div>

                                <div id="cardfuns" style={{ marginLeft: "-11%" }}>
                                    <MuiThemeProvider theme={theme}>
                                        {this.state.array.map(res => {
                                            return (
                                                <div id="addfun1ss">
                                                    {localStorage.getItem('cartID') === res.id ?
                                                        <MuiThemeProvider theme={theme3}>
                                                            <Card className="cardfun1s">

                                                                <div id="addfuns">SELECTED</div>

                                                            </Card>
                                                        </MuiThemeProvider>
                                                        :
                                                        <MuiThemeProvider theme={theme2}>
                                                            <Card className="cardfun1s">

                                                                <div id="addfuns">ADD TO CART</div>
                                                            </Card>
                                                        </MuiThemeProvider>}
                                                    <MuiThemeProvider theme={theme1}>
                                                        <Card style={{ position: "relative", padding: "5px" }} className="cardfun2">
                                                            <div style={{ marginLeft: "8%", marginRight: "3%", marginTop: "2%" }}>
                                                                <div id="addfun2">price: ${res.price} per month</div>
                                                                <div style={{ color: "blue", display: "flex", marginTop: "2%" }}>{res.name}</div>
                                                                <div style={{ display: "flex", flexDirection: "row", marginTop: "2%" }}>
                                                                    {/* <FiberManualRecordIcon fontSize="small" /> */}
                                                                    <div id="addfun3">.</div>
                                                                    <div id="addfun4">${res.price}/month</div>
                                                                </div>
                                                                <div style={{ display: "flex", flexDirection: "row", marginTop: "2%" }}>
                                                                    {/* <FiberManualRecordIcon fontSize="small" /> */}
                                                                    <div id="addfun3">.</div>
                                                                    <div id="addfun4">{res.description}</div>
                                                                </div>
                                                            </div>
                                                        </Card>
                                                    </MuiThemeProvider>

                                                </div>

                                            )
                                        })}

                                    </MuiThemeProvider>
                                </div>

                            </div>
                            : null}



                    </div>
                </div>
            </MuiThemeProvider>
        );
    }


}
export default Login