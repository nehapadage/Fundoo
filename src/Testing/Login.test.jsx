import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
// import { mount, shallow, render } from â€˜enzyme';
import Login from '../Components/Login'

// describe('Login', () => {
//     it('should render correctly in "debug" mode', () => {
//       const component = shallow(<Login debug />);
    
//       expect(component).toMatchSnapshot();
//     });
//   });

  describe('Login component tests', () => {

    const wrapper = shallow(<Login />);
    
    test('snapshot renders', () => {
      const component = renderer.create(<Login />);
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

    // it('login button is clicked', () => {
    //     const wrapper = shallow(<Login />);
    //     const loginBtn = wrapper.find('button.loginButton');
    //     loginBtn.simulate('click');
    //     const text = wrapper.find('e').text();
    //     expect(text).toEqual('@');
    //   });
  });

  