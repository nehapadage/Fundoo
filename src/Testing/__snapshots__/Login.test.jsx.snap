// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component tests should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`Login component tests snapshot renders 1`] = `
<div
  className="mainBody"
>
  <div
    className={null}
  >
    <div
      className="adminMainDiv"
    >
      <div
        id="fundoo"
      >
        <label
          id="flabel"
        >
          F
        </label>
        <label
          id="ulabel"
        >
          u
        </label>
        <label
          id="nlabel"
        >
          n
        </label>
        <label
          id="dlabel"
        >
          d
        </label>
        <label
          id="olabel"
        >
          o
        </label>
        <label
          id="o2label"
        >
          o
        </label>
      </div>
      <div>
        <div
          id="login"
        >
          Login 
        </div>
        <div
          id="text"
        >
          Use your Fundoo Account
        </div>
      </div>
      <div
        className="textfields"
      >
        <div
          className="emailIdField"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
            errorText=""
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
              data-shrink={false}
              htmlFor="EmailPass"
              id="EmailPass-label"
            >
              Email Id
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="email"
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="EmailPass"
                name="email"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="email"
                value=""
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-60 MuiOutlinedInput-notchedOutline"
                style={
                  Object {
                    "paddingLeft": 8,
                  }
                }
              >
                <legend
                  className="PrivateNotchedOutline-legend-61"
                  style={
                    Object {
                      "width": 0.01,
                    }
                  }
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "&#8203;",
                      }
                    }
                  />
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          style={
            Object {
              "color": "red",
              "fontSize": 12,
            }
          }
        >
          
        </div>
        <div
          className="PasswordField"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
              data-shrink={false}
              htmlFor="EmailPass"
              id="EmailPass-label"
            >
              Password
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="current-password"
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="EmailPass"
                name="password"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="password"
                value=""
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-60 MuiOutlinedInput-notchedOutline"
                style={
                  Object {
                    "paddingLeft": 8,
                  }
                }
              >
                <legend
                  className="PrivateNotchedOutline-legend-61"
                  style={
                    Object {
                      "width": 0.01,
                    }
                  }
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "&#8203;",
                      }
                    }
                  />
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          style={
            Object {
              "color": "red",
              "fontSize": 12,
            }
          }
        >
          
        </div>
        <div>
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-text"
            disabled={false}
            id="forgotPass"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              Forgot Password?
            </span>
          </button>
        </div>
        <div
          id="text"
        >
          Not your computer? Use Guest mode to sign in privately.
        </div>
        <div>
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-text"
            disabled={false}
            id="learnMore"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              Learn more
            </span>
          </button>
        </div>
        <div
          id="log"
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-text"
            disabled={false}
            id="createAccount"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              Create account
            </span>
          </button>
          <div>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              id="loginButton"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Login
              </span>
            </button>
          </div>
          <div />
        </div>
      </div>
    </div>
  </div>
</div>
`;
